services/blockchain-connector/shared/sdk/schema.js

```diff
@@ -51,17 +52,16 @@ const getBlockAssetDataSchemaByModule = _module => {
 const getDataSchemaByEventName = eventName => {
```

Comment:
Please update the method as discussed offline to cache the events by module and eventName instead.